person.rb

```diff
@@ -53,10 +53,3 @@ def correct_name
     @nameable.correct_name[0...10]
   end
 end
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227525259-2a6a08a1-aee4-435c-ac36-4dbb9ba1d9c7.png)

**[OPTIONAL]**

- [ ] It would be great if you delete your Live Demo section for now to make your readme file professional and useful information. Once you would have a live link then you can add this Live Demo section again with a working live link 👍